index.js

```diff
@@ -3,10 +3,21 @@ const path = require('path');
 const electron = require('electron');
 const unusedFilename = require('unused-filename');
 const pupa = require('pupa');
+const extList = require('ext-list');
 
 const app = electron.app;
 const shell = electron.shell;
 
+function getFilenameFromMime(name, mime) {
+	const ext = Object.keys(extList()).filter(x => extList()[x] === mime);
+
+	if (ext.length !== 1) {
+		return name;
+	}
```

Comment:
Thanks for pointing me to `mime-db`, great resource.

I already have a method for that, https://github.com/kevva/ext-name#extnamemimemimetype. The reason for using `ext-list` directly in here is to sort out possible duplicates. What I could do is add an option for getting only unique extensions, or a method that returns an array instead of the first matching object.